CODEOWNERS

```diff
@@ -9,7 +9,8 @@ firestore.rules andras@socialincome.org michael@socialincome.org
 package.json andras@socialincome.org michael@socialincome.org rene@socialincome.org
 storage.rules andras@socialincome.org michael@socialincome.org
 
-admin/ andras@socialincome.org michael@socialincome.org dev@socialincome.org
-functions/ andras@socialincome.org michael@socialincome.org dev@socialincome.org
-shared/ andras@socialincome.org michael@socialincome.org dev@socialincome.org
+admin/ andras@socialincome.org michael@socialincome.org sandino@socialincome.org
+functions/ andras@socialincome.org michael@socialincome.org sandino@socialincome.org
+shared/ andras@socialincome.org michael@socialincome.org sandino@socialincome.org
 
+.md/ sandino@socialincome.org andras@socialincome.org michael@socialincome.org
```

Comment:
@ssandino That except a folder `.md/`.

I think you have to check the example here regarding file extensions.

Reply:
[do you want to get these emails too?](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file)